ui/app/styles/components/selectable-card.scss

```diff
@@ -67,6 +67,11 @@
     font-weight: 500;
     line-height: 1.33;
   }
+
+  .vlt-table {
+    max-height: 240px;
```

Comment:
I think we need a min-height here, as if there are no know addrs then it looks a bit funny.  Also, there doesn't appear to be the empty state showing anymore.  Here's a screenshot where I expanded the min-height so you could see it.
![image](https://user-images.githubusercontent.com/6618863/83891263-20cc6880-a70a-11ea-8944-710e0b7eabca.png)